distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/common/Library.java

```diff
@@ -64,8 +64,10 @@ public static String getSimpleXmlNoDec(String uri) {
                 + "        <value>2013-04-18T10:50:27.371-07:00</value>\n" + "    </dateTime>\n"
                 + "    <dateTime name=\"modified\">\n"
                 + "        <value>2013-04-18T10:50:27.371-07:00</value>\n" + "    </dateTime>\n"
-                + "    <base64Binary name=\"thumbnail\">\n" + "        <value>CA==</value>\n"
-                + "    </base64Binary>\n" + "    <stringxml name=\"metadata\">\n"
+                + "    <base64Binary name=\"thumbnail\">\n" 
+                + "        <value>/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAALAAgDASIAAhEBAxEB/8QAFgABAQEAAAAAAAAAAAAAAAAAAAUH/8QAJBAAAAQFAwUAAAAAAAAAAAAAABETFAEDBBIVAgUGISJRYZP/xAAVAQEBAAAAAAAAAAAAAAAAAAADBf/EABkRAAMAAwAAAAAAAAAAAAAAAAABAwISIv/aAAwDAQACEQMRAD8A3fRvVTHmTS+RjkUfbolC+fUBDhR0zXMN5WUyibqyClrhMj8WdoAtynSGLfJ//9k=</value>\n"
+                + "    </base64Binary>\n" 
+                + "    <stringxml name=\"metadata\">\n"
```

Comment:
I'm surprised this isn't causing a failure. Usually with Windows we need to use either `System.lineSeparator()` or a format string with the `%n` code. If there is just a test for equality then it should be fine regardless of system.